GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/Client/TestRailClientWrapper.cs

```diff
@@ -130,6 +130,20 @@ public IEnumerable<Case> GetCases(ulong projectId, ulong suiteId)
             return result.Payload;
         }
 
+        /// <summary>
+        /// Moves feature files to new section
+        /// </summary>
+        /// <param name="newSectionId">Id of destination section</param>
+        /// <param name="caseIds">collection of feature file id's</param>
+        public void MoveCases(ulong newSectionId, IEnumerable<ulong> caseIds)
+        {
+            var policy = CreateResultHandlerPolicy<Result>();
+            var result = policy.Execute(()=>
+                _testRailClient.MoveCases(newSectionId, caseIds));
+            ValidateRequestResult(result);
+            Log.Info($"Cases {string.Join(", ", caseIds)} moved to section {newSectionId}");
```

Comment:
To be in a one style 
```suggestion
            Log.Info($"Moved: {string.Join(", ", caseIds)} to section {newSectionId}");
```

Reply:
Okay, but let's make it "Move cases: ...", because TestRail API is able to move other entities

Reply:
Ok